posix.mak

```diff
@@ -113,16 +113,16 @@ endif
 DDOC_VARS_STABLE=\
 	  DOC_OUTPUT_DIR="${DOC_OUTPUT_DIR}/phobos" \
 	  STDDOC="$(addprefix $(PWD)/, $(STD_DDOC))" \
-	  DMD="$(DMD_STABLE)" \
-	  DRUNTIME_PATH="${DRUNTIME_DIR}" \
+	  DMD="$(realpath $(DMD_STABLE))" \
```

Comment:
It's what we've used during the previous years, but I could use make's [abspath](https://www.gnu.org/software/make/manual/make.html#index-file-name_002c-abspath-of) if that helps.